numpy/version.py

```diff
@@ -12,4 +12,29 @@
 release = 'dev0' not in version and '+' not in version
 short_version = vinfo['version'].split("+")[0]
 
-del get_versions, vinfo
+
+def make_version_tuple():
```

Comment:
```suggestion
def make_version_tuple() -> tuple[int, int, int]:
```
Could you add a return type annotation for the function?
`np.version` is one of the rare few module that contains inline annotations, rather than a dedicated stub file (as it is a rather simple module, annotation wise).